packages/trpc/server/routers/viewer/eventTypes/getUserEventGroups.handler.ts

```diff
@@ -59,9 +61,29 @@ export const getUserEventGroups = async ({ ctx, input }: GetByViewerOptions) =>
   const profileProcessor = new ProfilePermissionProcessor();
   const profiles = profileProcessor.processProfiles(eventTypeGroups, teamPermissionsMap);
 
+  const permissionCheckService = new PermissionCheckService();
+
+  const teamPermissionsArray = await Promise.all(
+    filteredEventTypeGroups
+      .map((group) => group.teamId)
+      .filter((teamId): teamId is number => teamId !== null && teamId !== undefined)
+      .map(async (teamId) => {
+        const canCreateEventType = await permissionCheckService.checkPermission({
+          userId: user.id,
+          teamId: teamId,
+          permission: "eventType.create",
```

Comment:
I think we should use constants here 

  const CREATE_EVENT_TYPE_PERMISSION = "eventType.create";

Reply:
This is already fully typed, so if you put `evnetType.create`, TypeScript will throw an error :)